xbmc/video/VideoDatabase.cpp

```diff
@@ -271,6 +274,10 @@ void CVideoDatabase::CreateAnalytics()
 
   m_pDS->exec("CREATE INDEX ix_rating ON rating(media_id, media_type(20))");
 
+  m_pDS->exec("CREATE INDEX ix_uniqueid1 ON uniqueid(uniqueid_id, media_id, media_type(20))");
+  m_pDS->exec("CREATE INDEX ix_uniqueid2 ON uniqueid(uniqueid_id, media_id, value(20))");
+  m_pDS->exec("CREATE INDEX ix_uniqueid3 ON uniqueid(media_type(20), value(20))");
```

Comment:
@Montellese I have to create only 2 right? Because if I'm not wrong in both sqlite and mysql a index with `media_id`, `media_type` and `type` will work also for only with `media_id` and `media_type`